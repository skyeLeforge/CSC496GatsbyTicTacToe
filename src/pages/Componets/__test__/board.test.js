import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"

import Board from "../board.js"

describe("Board", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<Board squares ={Array(9).fill(null)} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
)