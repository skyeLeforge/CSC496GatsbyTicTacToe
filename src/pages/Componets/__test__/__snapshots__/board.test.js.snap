// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board renders correctly 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
    <button
      className="square"
      data-testid="secret-test-id"
      onClick={[Function]}
    />
  </div>
</div>
`;
